traveler_app/build.gradle

```diff
@@ -26,46 +26,44 @@ android {
 
     buildTypes {
         release {
+            //TODO add signingConfig. doesn't work without it
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             buildConfigField("String", "TRAVELER_API_KEY", "\"" + getReleaseTravelerAPIKey() + "\"")
             buildConfigField("String", "GOOGL_SIGN_IN_CLIENT_ID", "\"" + getReleaseGoogleSignInClientID() + "\"")
+            buildConfigField "String", "AUTH_ENDPOINT_URL", "\"https://9th3dtgfg3.execute-api.ca-central-1.amazonaws.com/dev\""
+            buildConfigField "String", "SDK_ENDPOINT_URL", "\"https://traveler.guestlogix.io/v1\""
         }
 
         debug {
             minifyEnabled false
             multiDexEnabled true
             buildConfigField("String", "TRAVELER_API_KEY", "\"" + getDebugTravelerAPIKey() + "\"")
             buildConfigField("String", "GOOGL_SIGN_IN_CLIENT_ID", "\"" + getDebugGoogleSignInClientID() + "\"")
+            buildConfigField "String", "AUTH_ENDPOINT_URL", "\"https://r3p9qio0x7.execute-api.ca-central-1.amazonaws.com/dev\""
+            buildConfigField "String", "SDK_ENDPOINT_URL", "\"https://traveler.dev.guestlogix.io/v1\""
+        }
+
+        staging {
+            initWith debug
+            buildConfigField "String", "AUTH_ENDPOINT_URL", "\"https://hklkg7c974.execute-api.ca-central-1.amazonaws.com/dev\""
+            buildConfigField "String", "SDK_ENDPOINT_URL", "\"https://traveler.rc.guestlogix.io/v1\""
+            matchingFallbacks = ['debug', 'release']
+        }
+
+        prod {
+            //TODO remove in future when 'release' build type will be properly configured
+            initWith debug
+            buildConfigField "String", "AUTH_ENDPOINT_URL", "\"https://9th3dtgfg3.execute-api.ca-central-1.amazonaws.com/dev\""
+            buildConfigField "String", "SDK_ENDPOINT_URL", "\"https://traveler.guestlogix.io/v1\""
+            matchingFallbacks = ['debug', 'release']
         }
```

Comment:
Yeah, I explained this in my comment above.

Firslty I used build flavours. You can see this in this commit:
https://github.com/Guestlogix/traveler-android/pull/421/commits/8036fa613988864ac74c634d0db7853801238211
(used flavours 'dev', 'rc' and 'prod')

But I guess that CI tries to build application using 'debug' build variant. 
After adding flavours, we get 'devDebug', 'devRelease', 'rcDebug', 'rcRelease', 'prodDebug' and 'prodRelease' build variants. But an old one 'debug' build variant is not exist anymore.
That's why checks on CI have not passed. Is simply was unable to build app, because 'debug' variant was not found.

That's why in second commit in this branch I moved configs to build types instead of product flavours. Otherwise CI not pass checks.

But I fully agree with you, that using flavours for defining endpoints is better way instead fo using build types. My first commit was exact like this.

Can you please modify CI settings to use 'devDebug' variant instead of 'debug'? Then I can revert to use flavours instead of build types.

Reply:
yea ci settings should be changed. can you find the failed build in CI panel and comment it here so i can ask Omar to take a look.

Reply:
fixed those settings....try again?

Reply:
Also please squash all your commits into a single one. and take into account that we always rebase, we never merge